src/shared/services/commandInterpreterHelper.ts

```diff
@@ -505,32 +501,16 @@ const availableCommands = [
   },
   {
     name: 'guide',
-    match: (cmd: any) => /^guide(\s|$)/.test(cmd),
-    exec(action: any, put: any, store: any) {
-      const guideName = action.cmd.substr(':guide'.length).trim()
-      if (!guideName) {
-        put(resetGuide())
-        put(open('guides'))
+    match: (cmd: string) => /^guide(\s|$)/.test(cmd),
+    exec(action: ExecuteSingleCommandAction, dispatch: Dispatch<Action>) {
```

Comment:
This action is not necessarily `ExecuteSingleCommandAction`, it could also be a SYSTEM_COMMAND_QUEUED action or some other action (such as on line 257 where the action is created without an action creator). For now I'm not sure what the best way to handle this is. I think we can leave it like it is now or change the type to be {cmd: string} instead. Anyway, this problem will go away when commandInterpreterHelper is fully typed

Reply:
@OskarDamkjaer - I think I'll create a generic type/interface `CommandAction`. Then, any types can extend this one. And we can use this generic action in this file.
Will do in subsequent PR.